src/config/yaml_baking_conf_parser.py

```diff
@@ -176,6 +178,18 @@ def validate_min_delegation_amt(self, conf_obj):
                 )
             )
 
+    def validate_min_payment_amt(self, conf_obj):
+        if MIN_PAYMENT_AMT not in conf_obj:
+            conf_obj[MIN_PAYMENT_AMT] = 0
+            return
+
+        if not self.validate_non_negative_int(conf_obj[MIN_PAYMENT_AMT]):
```

Comment:
It is the first time I become aware of the "validate_non_negative_int" function. That thing feels wrong... but not your cup of tea

Reply:
Ok, I'll mark this as resolved in this case.